src/Mod/Part/App/FCRepAlgoAPI_BooleanOperation.cpp

```diff
@@ -0,0 +1,56 @@
+/***************************************************************************
```

Comment:
This is the old license text with the wrong address. For the new text look at https://github.com/FreeCAD/FreeCAD/blob/main/src/Gui/TaskView/TaskImage.h
In order to fit the copyright it's best to add extra spaces to the text underneath.

Reply:
fixed